drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -545,7 +545,7 @@ static int stm32_clock_control_init(struct device *dev)
 	/* Main PLL configuration and activation */
 	LL_RCC_PLL_SetSource(LL_RCC_PLLSOURCE_HSI);
 
-	pllsrc_clock = HSI_VALUE;
+	pllsrc_clock = HSI_VALUE / CONFIG_CLOCK_STM32_HSI_DIVISOR;
```

Comment:
This is correct but, I wonder if we can just remove `pllsrc_clock` and replace by use of constant `PLLSRC_FREQ` which is already defined conditionally:
```
/* Choose PLL SRC */
#ifdef CONFIG_CLOCK_STM32_PLL_SRC_HSI
#define PLLSRC_FREQ  ((HSI_FREQ)/(CONFIG_CLOCK_STM32_HSI_DIVISOR))
#elif defined(CONFIG_CLOCK_STM32_PLL_SRC_CSI)
#define PLLSRC_FREQ  CSI_FREQ
#elif defined(CONFIG_CLOCK_STM32_PLL_SRC_HSE)
#define PLLSRC_FREQ  HSE_FREQ
#else
#define PLLSRC_FREQ 0
#endif
```

Reply:
It makes a perfect sense. I will update the PR.

Reply:
Maybe even whole `get_vco_input_range()` function should be changed to macro and warn about wrong settings in compile time...

Reply:
go ahead!

Reply:
You're right, that makes sense I missed that when making the original PR. Using the constants will be best !

Reply:
Then you can remove any ```pllsrc_clock``` occurance has it's no longer needed. I added it only to  compute the VCO input range.

Reply:
> Then you can remove any pllsrc_clock occurance
I thought I have done it, thanks for comment. Fixed now